crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -438,6 +653,86 @@ TEST(ServiceIndicatorTest, AESGCM) {
       encrypt_output, &out_len, sizeof(encrypt_output), nonce, EVP_AEAD_nonce_length(EVP_aead_aes_128_gcm()),
           kPlaintext, sizeof(kPlaintext), nullptr, 0));
   ASSERT_EQ(approved, AWSLC_NOT_APPROVED);
+
+  // Call non-approved external IV usage of AES-GCM 192 bit key size. (192 is
```

Comment:
Also add test for the regular gcm algorithms that are not approved, not only the 192-bit one?